packages/mds-trip-processor/src/proc-trip.ts

```diff
@@ -123,11 +121,11 @@ export async function tripProcessor() {
       )
 
       results.map(response => {
```

Comment:
If you are using `map` and not assigning the result to anything then `forEach` is probably a better choice.

Reply:
My understanding is that map is faster than forEach even though we don;t use the returned result. Regardless, I'm opening a PR to optimize the caching which will eliminate this logic